### ISSUE TITLE
Make local orderID optional 

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#254]
The reason why orders entered as `xucli placeorder BTC/LTC 1 2` don't show via `./xucli getinfo` is because they are missing the orderID param: `xucli placeorder BTC/LTC **100** 1 2`

We definitely want the orderID via gRPC, but I am sure most use cases via CLI don't require a orderID.

This issue is about making this xucli param optional.

**Comment by kilrau** [comment]
Yep, exactly generate one. And I can look it up via get orders or sth. Then use it to cancel.

**Comment by kilrau** [comment]
Oh this was supposed to be alpha

**Comment by sangaman** [comment]
Just to note, the `placeorder` grpc call should probably return the localId if the order gets added to the orderbook, otherwise it can just return nothing to indicate that the order has been fully matched or discarded.

**Comment by kilrau** [comment]
Good idea! `{Order [id] placed successfully!}`

**Comment by sangaman** [comment]
Yeah, doesn't even need to be in `{}` I'd say, save that for JSON output.
